templates/index.html

```diff
@@ -1,13 +1,38 @@
-{% extends "layout/default.html" %}
+{% extends "_layout.html" %}
 
 {% block title %}Search assets{% endblock %}
 
 {% block content %}
-    {% if assets %}
-        <h1>Assets containing '{{ query }}'</h1>
-        {% include "includes/list.html" %}
-    {% elif query %}
-        <h1>Assets containing '{{ query }}'</h1>
-        <p>No assets found.</p>
-    {% endif %}
+    <div class="row">
+        <div class="col-12">
+        <form action="/" class="p-form p-form--inline">
+            <div class="p-form__group">
+                <div class="p-form__control u-clearfix">
+                    <input type="search" class="p-form__control" name="q" placeholder="Search" value="{{ query }}">
```

Comment:
You should use the search pattern: https://docs.vanillaframework.io/patterns/search-box/

Reply:
Can I use that pattern and have another input in-line?

Reply:
Aaaaaaant help me frontend!

Reply:
Does the search button search and filter at the same time?

Reply:
Yeah, just sends an empty `type` before anything is set.